generators/angular/templates/src/main/webapp/app/entities/_entityFolder_/list/_entityFile_.component.html.ejs

```diff
@@ -88,11 +90,14 @@
 <%_ for (const relationship of relationships.filter(rel => !rel.otherEntityIsEmbedded)) { _%>
   <%_ if (relationship.persistableRelationship && (!relationship.collection || paginationNo)) {
     const fieldName = "." + relationship.otherEntityField;
+    const { fieldSupportsSortBy } = relationship.relatedField;
```

Comment:
Correct me if I am wrong, we only support sorting by relationship id, not relationship label on backend side.
```suggestion
    const { fieldSupportsSortBy: relationshipPkSupportsSortBy } = relationship.primaryKey.fields[0];
```

Reply:
No, we actually support sorting by the label, which even makes sense as it would otherwise be really confusing in the UI after applying the sort. 
But that only works if the label field itself is sortable, i.e. not Transient like the mapstructExpression field.